config/config.go

```diff
@@ -204,6 +205,7 @@ func DefaultConfigLocalnet() *Config {
 	conf.Network.DefaultPort = 0
 	conf.Network.ForcePrivateNetwork = true
 	conf.Network.EnableMdns = true
+	conf.Network.PeerStorePath = "localnet-peers.json"
```

Comment:
The working directory is all about one instance. 
If we want to to follow this, then we should have `testnet-config`, `testnet-wallet`, etc